walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -20,16 +20,23 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns HTML that contains the page view count. */
-@WebServlet("/page-views")
+/** Servlet(@webservlet) that returns HTML that contains the page view count when triggered by client request. */
+
+@WebServlet("/page-views") 
 public class PageViewServlet extends HttpServlet {
 
   private int pageViews = 0;
-
+  //do get function below runs whenever a client sends a GET request to the servlet URL
```

Comment:
When referring to a function in the comments, ensure the string matches (ie, "do get" != "doGet").

Specifically when documenting a function, consider using JSDoc formatting:
https://developers.google.com/closure/compiler/docs/js-for-compiler

But if you don't use JSDoc, please use a **consistent** block comment style:
https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style